sdk/identity/identity/src/msal/nodeFlows/nodeCommon.ts

```diff
@@ -121,7 +121,7 @@ export abstract class MsalNode extends MsalBaseUtilities implements MsalFlow {
       authorityHost: authority
     });
 
-    let clientCapabilities: string[] = ["CP1"];
+    let clientCapabilities: string[] = ["cp1"];
```

Comment:
If one goes to the service documentation, this value appears in lowercase: https://docs.microsoft.com/azure/active-directory/develop/claims-challenge

This change has no effect other than helping me have a more 1:1 comparison between our recordings and our docs.